wgpu/gui/base.py

```diff
@@ -16,6 +16,9 @@ class WgpuCanvasInterface(base.GPUCanvasContext):
 
     # NOTE: It is not necessary to actually subclass this class.
 
+    # Whether to present to a surface or to a texture
+    _PRESENT_TO_SURFACE = True
```

Comment:
This could be a property that checks if _present() raises NotImplementedError. Would save a bit of bookkeeping :)